homeassistant/components/flume/sensor.py

```diff
@@ -152,7 +156,11 @@ def update(self):
             self._available = False
             return
         _LOGGER.debug("Successful update of flume sensor: %s", self._name)
-        self._state = self._flume_device.value
+        self._state = self._flume_device.values["current_interval"]
+        self._attributes = {
+            k: round(self._flume_device.values[k], 1)
```

Comment:
Please set the attribute keys explicitly. We want to check what they are and the formatting.

Reply:
Added in the latest push.